larray/core/session.py

```diff
@@ -259,18 +280,30 @@ def save(self, fname, names=None, engine='auto', overwrite=True, display=False,
         display : bool, optional
             Whether or not to display which file is being worked on. Defaults to False.
 
+        Notes
+        -----
+        When session's objects are saved in CSV files, all Axis (Group) objects are saved in the same CSV file
```

Comment:
you could start by saying that array objects are saved one per CSV file/excel sheet.